src/components/App.js

```diff
@@ -15,13 +16,25 @@ import ProductDetail from './ProductDetail/ProductDetail';
 import { cleanFetchedData } from '../helperFunctions';
 
 function App() {
-  const [shoppingBag, setShoppingBag] = useState([]);
-  const { loading, error, data } = useQuery(GET_ALL_ITEMS);
+  const [cookies, setCookie] = useCookies(['shoppingBag']);
+  const [shoppingBag, setShoppingBag] = useState([])
+    
+  const { loading, error, data } = useQuery(GET_ALL_ITEMS)
   const [totalPrice, setTotalPrice] = useState(0);
   const [items, setItems] = useState([]);
   const [itemsForDisplay, setItemsForDisplay] = useState([]);
   const [successMessage, setSuccessMessage] = useState('');
 
+  useEffect(() => {
```

Comment:
Very cool! Thanks for handling this sick feature!

Reply:
+1